eth/tracers/tracer.go

```diff
@@ -670,7 +670,7 @@ func (jst *JsTracer) CaptureStart(env *vm.EVM, from common.Address, to common.Ad
 	jst.ctx["to"] = to
 	jst.ctx["input"] = input
 	jst.ctx["gas"] = gas
-	jst.ctx["gasPrice"] = env.Context.GasPrice
+	jst.ctx["gasPrice"] = env.GasPrice
```

Comment:
I think tracer doesn't need gasPrice. Do we need it?

Reply:
I think tracer need this field. This line was submitted #558 and also exist in geth.